doc/source/whatsnew/v0.24.0.rst

```diff
@@ -362,6 +362,7 @@ See the :ref:`Advanced documentation on renaming<advanced.index_names>` for more
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
+- Added support for %G strptime directive in parsing datetimes
```

Comment:
* Reference the issue number
* I think this description is a little too technical for our general end user.  Can we describe the enhancement without referencing the directive?

Reply:
- Done
- Did change it into 'Added support for ISO week year format when parsing datetimes using `to_datetime`' if that works better

Reply:
Much better.  Thanks!